cotton-datapack/src/main/java/io/github/cottonmc/cotton/datapack/CottonInitializer.java

```diff
@@ -1,4 +1,4 @@
-package io.github.cottonmc.cotton;
+package io.github.cottonmc.cotton.datapack;
```

Comment:
`CottonInitializer`s aren't really datapack-related, leaving it in the root package might be good. I don't mind having it in the datapack module, though.

Reply:
There’s no default module, so it is in datapack

Reply:
Yep, I'm suggesting that it should go to the root package (`io.g.cmc.cotton`) in the datapack module.

Reply:
Would probably only cause confusion

Reply:
True.